### ISSUE TITLE
Custom F-Droid repositories (FUTO)

### ISSUE THREAD

**Issue by martin-braun** [ImranR98/Obtainium#1671]
**Describe the feature**
I would like to have access to the FUTO F-Droid repository: https://app.futo.org/fdroid/repo/

I like their apps, but they don't seem to be interested in using the default F-Droid repository. However, them having only 4 apps makes it hard to justify. The source list is already very long. So, instead of suggesting FUTO directly, I would like to suggest to support adding third party F-Droid repositories instead.

I also want to be clear: I was able to use the HTML Source method for FUTO Voice Input, one of their apps. But, unfortunately, some of their apps, like Grayjay, don't show the current version in their HTML page. Their way of distribution is not coherent, but through F-Droid - by design - it is.

**Describe alternatives you've considered (if applicable)**
As said, for some apps the HTML Source method works, but for some, it doesn't due to missing version information. Also the download link doesn't feature a version.

**Additional context**
My primary suggestion is to add the ability to add custom F-Droid source repositories, not FUTO directly.

Thank you!

**Comment by DwainZwerg** [comment]
It already works. Please read the documentation: https://github.com/ImranR98/Obtainium/wiki#f-droid-third-party-repo

**Comment by ImranR98** [comment]
Yep, this is already supported.

**Comment by martin-braun** [comment]
@DwainZwerg @ImranR98 Thanks, yes this is correct, I got it to work, but here is where the confusion was kicking in due to trying to approach the functionality in a intuitive way:

- Tab "Add app" lists "F-Droid Third-Party Repo (Searchable)" under "Supported Sources", so I assumed, I can add this somewhere, however
- Search (Some Sources Only) would not allow me to pick F-Droid Third-Party Source
- As the documentation points out, I can search my third party source in "Import/Export"/"Search Source"/"F-Droid Third-Party Repo", this indeed works, but
- My third party source will not be remembered. Say I want to add another app, I still need to grab the URL and paste it in

My intuition was searching for a way to add a third party repo, so that "Add App"/"Search (Some Sources Only)" would also allow me to pick "F-Droid Third-Party Repos" (missing) to search through my own repos that I added.

I think this would be a great addition to avoid so much confusion, but I'm happy this is working now.
